File: contest/typical90N/typical90N.cpp
#include <bits/stdc++.h>
using namespace std;

using ll = long long;

int main(){
    // それぞれにソートして並列で見ていく感じ
    int N;
    cin >> N;
    vector<ll> A(N, 0), B(N, 0);
    for (int i=0; i<N; i++) {
        cin >> A[i];
    }
    for (int i=0; i<N; i++) {
        cin >> B[i];
    }
    sort(A.begin(), A.end());
    sort(B.begin(), B.end());

    ll ans = 0;
    for (int i=0; i<N; i++) {
        ans += abs(A[i]-B[i]);
    }
    cout << ans << endl;
    return 0;
}

File: contest/typical90N/typical90N.py
import sys

input = lambda: sys.stdin.readline().rstrip()

N = int(input())
A = list(map(int, input().split()))
B = list(map(int, input().split()))

A.sort()
B.sort()
print(sum([abs(a - b) for a, b in zip(A, B)]))
